docs/wiki/sensestudio.md

```diff
@@ -0,0 +1,87 @@
+SenseStudio is our one-stop, web-based platform to quickly get you started with
+developing your own classifier. On this page you will find some tips and more 
+detailed instructions on making the most of the tools we provide.
+
+---
+
+## Setting up your project
+
+SenseStudio can be launched by running the `sense_studio.py` script and opening `http://127.0.0.1:5000/`
+in your local browser.
+
+```commandline
+PYTHONPATH=./ python tools/sense_studio/sense_studio.py
+```
+
+On the welcome page you have the option of creating a new project by providing a project name and 
+path, or, importing an existing project (previously created with SenseStudio).
+
+Projects are initialised with the following structure:
+```
+/path/to/your/dataset/
+├── videos_train
+├── videos_valid
+└── project_config.json
+```
+
+The project config stores project details and settings such as time created, classes, video
+recording parameters, etc.
+
+---
+
+## Creating New Classes
+
+New classes can be added from the `Project Details` tab. Creating a
+new class automatically updates the video folder structure and project config. 
+
+**NOTE**: Class names can be edited directly on SenseStudio from the `Project Details` page. 
+Updating it in `project_config.json` may break SenseStudio if the relevant folder names are not
+updated.
```

Comment:
Not sure if we should even hint at the possibility of manually editing `project_config.json` :grimacing:

Reply:
We can remove the reference :p